ttnn/cpp/ttnn/operations/eltwise/binary/device/element_wise_multi_core_program_factory.cpp

```diff
@@ -272,6 +272,9 @@ BinaryDeviceOperation::ElementWiseMultiCore::cached_program_t BinaryDeviceOperat
     tt::DataFormat dst_cb_data_format = tt_metal::datatype_to_dataformat_converter(output.get_dtype());
     uint32_t dst_single_tile_size = tt_metal::detail::TileSize(dst_cb_data_format);
 
+    tt::DataFormat interim_cb0_format = src0_cb_data_format;
+    tt::DataFormat interim_cb1_format = src1_cb_data_format;
+
```

Comment:
Why was creating this extra variable necessary?

Reply:
@eyonland For some op we need intermediate buffers for both the operand and it need to be on format as its source. For logaddexc PCC issues these needs to be in bf16 format, hence to handle the generic cases created new variable and used it